server_installation/topics/operator/keycloak-realm-cr.adoc

```diff
@@ -0,0 +1,63 @@
+
+=== KeycloakRealm Custom Resource
+
+The {project_operator} allows application developers to represent {project_name} Realms as Custom Resources:
+
+.`KeycloakRealm` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakRealm
+metadata:
+  name: example-keycloakrealm
+  labels:
+    app: sso
+spec:
+  realm:
+    id: "basic"
+    realm: "basic"
+    enabled: True
+    displayName: "Basic Realm"
+  instanceSelector:
+    matchLabels:
+      app: sso
+```
+
+TIP: Note, that `instanceSelector` needs to match labels of an existing `Keycloak` Custom Resource.
+
+{project_operator} synchronizes all the changes made to the Custom Resource with a running {project_name} instance. No manual changes via {project_name} Admin Console are allowed.
```

Comment:
This two sentences should be annotated with `IMPORTANT:` too.

Reply:
I added a `NOTE`